syntax = "proto3";

message SetRoleRequest {
  string uuid = 1;
  string role = 2;
}

message SetRoleResponse {
  bool result = 1;
  string status = 2;
}

message CheckUserExistsRequest {
  string uuid = 1;
}

message CheckUserExistsResponse {
  bool result = 1;
}

service Auth {
  rpc SetRole (SetRoleRequest) returns (SetRoleResponse) {}
  rpc CheckUserExists (CheckUserExistsRequest) returns (CheckUserExistsResponse) {}
}